src/plugins/cams/client.py

```
@@ -0,0 +1,236 @@
+    HAS_CDSAPI = False
+    
+logger = logging.getLogger(__name__)
+
+# Load environment variables
+load_dotenv()
+
+
+class CAMSClient:
+    """Client for accessing CAMS data via Copernicus Data Store API."""
+    
+    def __init__(self, api_uid: Optional[str] = None, api_key: Optional[str] = None):
```

Ruff (0.12.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)